opal/runtime/opal_params_core.c

```diff
@@ -258,6 +258,7 @@ int opal_register_util_params(void)
         return OPAL_ERR_OUT_OF_RESOURCE;
     }
 
```

Comment:
Move the comment with the color cheatsheet together with the assignment? (and remove it from above at the declaration)

```suggestion

    /* Basic color options: 30=black, 31=red, 32=green,
     * 33=yellow, 34=blue, 35=magenta, 36=cyan, 37=white
     * https://en.wikipedia.org/wiki/ANSI_escape_code#Colors */
```